src/lib/transform.js

```diff
@@ -145,19 +145,29 @@ function transformApproversModel(prop) {
   return transformer;
 }
 
-function transformGrantModel(prop) {
+function transformGrantModel(prop, sortBy = null) {
```

Comment:
Can we add a comment here and where it's used explaining that passing sortBy will also mean not obtaining a distinct list?